src/data-acess/user.transfer.ts

```diff
@@ -0,0 +1,17 @@
+import { Sequelize } from "sequelize";
+import { UserModel } from "../interfaces/user.types";
+import { sequelizeUserModel } from "../models/user.model";
+
+export const createConnection = (url: string): Sequelize => {
+    return new Sequelize(url);
+}
+
+export const defineUsers = (connsection: Sequelize) => {
+    return connsection.define('User', sequelizeUserModel, {
+        timestamps: false
+    });
+}
+
```

Comment:
It's better to  give more meaningful names to the files. I'm not sure what this file is for. It seems that `defineUsers` is needed only for initial migration of DB to fill it with data.